migration/20220608-fix-broken-deprioritize-lcp-config.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import logging
+import os
+import sys
+
+from sqlalchemy import text
+from sqlalchemy.orm import Session
+
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+
+from core.model import production_session
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel("INFO")
+
+
+def execute_migration(db: Session) -> None:
+    db.execute(
+        text(
+            """
+UPDATE configurationsettings
+  SET value = 'false'
+  WHERE value = 'Do not de-prioritize'
+    """
+        )
+    )
+    db.execute(
+        text(
+            """
+UPDATE configurationsettings
+  SET value = 'true'
+  WHERE value = 'De-prioritize'
+    """
```

Comment:
We don't need to do all of our migrations using Python. This migration would be shorter and easier to understand if done in SQL:

```
-- The purpose of this migration is to fix any weird values that were inserted
-- into the configurationsettings table by the Admin API. Previous versions of
-- the code represented the LCP priority setting as an enum, and the Admin API
-- inserted the configuration labels rather than the enum values into the database.

UPDATE configurationsettings SET value = 'false' WHERE value = 'Do not de-prioritize';
UPDATE configurationsettings SET value = 'true' WHERE value = 'De-prioritize';
```

Reply:
Noted! I actually didn't realize we could include SQL files directly and they'd be executed. I saw other files, but I thought they were being opened from Python scripts.

Had I known, this would have been pure SQL...

Reply:
No worries. I figured this might be the case. As you suspected, the pure SQL migrations are indeed being opened by a python script. It's just that the script is part of the migration machinery, rather than something custom to a specific migration.